src/dispatching/dispatchers/filter/mod.rs

```diff
@@ -303,7 +303,7 @@ mod tests {
 
     use crate::{
         dispatching::{
-            dispatchers::filter::FilterDispatcher, updater::StreamUpdater,
+            dispatchers::filter::FilterDispatcher, updater::LongPollingUpdater,
```

Comment:
But there is no `LongPollingUpdater`... (and in my opinion, it should be called just "LongPolling")

Reply:
But how to name `WebhookUpdater`? Simply `Webhook`? It seems quite strange to me.

Reply:
Also, `LongPollingUpdater` is a short name compared to Spring's classes and interfaces.

Reply:
Yep. Just `Webhook`. We have no structures with similar names + they both are placed in `updater` module.

I hate java naming style, so spring means nothing :woman_shrugging:

Reply:
OK, I think using `updater::LongPolling` and `updater::Webhook` is normal.